cnf-certification-test/suite_test.go

```diff
@@ -56,8 +62,10 @@ const (
 	defaultClaimPath              = ".."
 	defaultCliArgValue            = ""
 	junitFlagKey                  = "junit"
+	serverModeFlag                = "server-mode"
 	TNFReportKey                  = "cnf-certification-test"
 	extraInfoKey                  = "testsExtraInfo"
+	defaultServerMode             = "false"
```

Comment:
instead of asking for a value, use only the presence of the flag as true, its absence as false

Reply:
the command still asks for a value. 
./run-cnf-suites.sh -s true
should be like:
./run-cnf-suites.sh -s

same for the cnf-certification-test.test executable and the server-mode flag

Reply:
i think is it ok to do that just from the ./run-cnf-suites.sh side?